k8s_helm_charts/zdm/templates/cdm.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: cdm
+  name: cdm-1
```

Comment:
why -1?

Reply:
Should be addressed in the latest commit.